transports/sftp/src/main/java/org/mule/transport/sftp/SftpMessageReceiver.java

```diff
@@ -170,6 +175,20 @@ protected boolean pollOnPrimaryInstanceOnly()
         return poolOnPrimaryInstanceOnly;
     }
 
+    @Override
+    public MuleMessage createMuleMessage(Object transportMessage) throws MuleException
```

Comment:
If you override createMuleMessage(Object message, String encoding) you don't need to replicate the encoding logic.